index.html

```diff
@@ -81,6 +81,51 @@ <h1>
 		</section>
 
 		<section class="works" id="portfolio">
+			<div class="popup-container">
+				<!-- TEMPLATE TO USE WITH JS -->
+				<!-- 	<div class="main-container">
+					<div class="content-container">
+						<div class="header">
+							<h1>title</h1>
+							<div id="btn-close" class="fas fa-times"></div>
+						</div>
+						<ul class="info">
+							<li>info</li>
+							<li>info</li>
+							<li>info</li>
+							<li>info</li>
+						</ul>
+						<div class="image-container">
+							<img
+								src="img/desktop/desktop-img1.jpg"
+								class="card-image"
+								alt=""
+							/>
+						</div>
+						<div class="container-secondary">
+							<p>
+								Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+								Reprehenderit, fuga?
+							</p>
+							<div class="skills-info">
+								<ul>
```

Comment:
- [ ] Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.